File: js/sumOfCubes.js
// Create a function that takes in an array of numbers and returns the sum of its cubes.
// https://edabit.com/challenge/XdAR3KohR5w7rjrFg

const getSum = (arr) =>
  arr.map((num) => Math.pow(num, 3)).reduce((acc, curr) => acc + curr, 0);

module.exports = getSum;

File: CSharp/SumOfCubes.cs
using System;
using System.Linq;

namespace CSharp
{
    // Create a function that takes in an array of numbers and returns the sum of its cubes.
    // https://edabit.com/challenge/gojyHs5xGwpFtZfWK
    public static class SumOfCubes
    {
        public static int GetSum(int[] arr) => arr.Select(num => (int)Math.Pow(num, 3)).Sum();
    }
}